### ISSUE TITLE
[Bug] Inaccessibility: duplicate IDs on checklist page

### ISSUE THREAD

**Issue by aarongoldenthal** [a11yproject/a11yproject.com#1441]
Thank you for reporting a bug. Please fill out the following information:

## Bug description

The [Checklist page](https://www.a11yproject.com/checklist/) contains four instances of duplicate IDs, a violation of [WCAG success criterion 4.1.1](https://www.w3.org/TR/WCAG20-TECHS/F77).

The [checklist title](https://github.com/a11yproject/a11yproject.com/blob/e8fcce9afa66492011ab7d7b25befcb0f50a40ab/src/_includes/checklist__task.njk#L20) renders the ID as a slugified version of the title. So, in [the checklists](https://github.com/a11yproject/a11yproject.com/blob/main/src/_data/checklists.json) any cases where `checkboxId` matches the slugified `title` there will be duplicate IDs on the page. 

The current issues are listed below, but with the current template implementation this could recur.

- [Remove invisible focusable elements](https://github.com/a11yproject/a11yproject.com/blob/e8fcce9afa66492011ab7d7b25befcb0f50a40ab/src/_data/checklists.json#L114)
- [Don't skip heading levels.](https://github.com/a11yproject/a11yproject.com/blob/e8fcce9afa66492011ab7d7b25befcb0f50a40ab/src/_data/checklists.json#L212)
- [ Remove seizure triggers.](https://github.com/a11yproject/a11yproject.com/blob/e8fcce9afa66492011ab7d7b25befcb0f50a40ab/src/_data/checklists.json#L389)
- [Remove horizontal scrolling.](https://github.com/a11yproject/a11yproject.com/blob/e8fcce9afa66492011ab7d7b25befcb0f50a40ab/src/_data/checklists.json#L540)

## Expected outcome

Page does not have a11y issues.

## Steps to reproduce

Open page (although in this case a Pa11y scan did flag this issue).

###  What device were you using when you encountered the bug?

Laptop computer

### What operating system were you using? What version is the operating system?

Win 10 21H2

### What web browser were you using? What version is the web browser?

Chrome 101.0.4951.67 

### Assistive technology and version (optional)

None

## Terms

- [x] I have read, and agree to follow this project's [Code of Conduct](https://www.a11yproject.com/code-of-conduct/)

**Comment by SaptakS** [comment]
> but with the current template implementation this could recur

I agree. I think updating the checkboxId with a suffix like `-checkbox` would solve the issue. But sadly that would mean that anyone who had their checklist state saved in localStorage would be removed. I think there's a workaround for that. I can check if the current id stored in localStorage + `-checkbox` is a valid element in the page, then I can update the localStorage and then mark the checkbox as checked.

@ericwbailey any thoughts?

**Comment by ericwbailey** [comment]
@SaptakS I like the suffix approach. I'm also of the school of thought that a discrete access issue is more important than hypothetical checklist efforts in flight. I personally don't mind breaking that if it means the page is now more accessible.

**Comment by SaptakS** [comment]
Cool. I will make a PR for this sometime this weekend.

**Comment by ericwbailey** [comment]
@SaptakS Thank you for that 🙏🏻
